README.md

```diff
@@ -1 +1,53 @@
-# hello_microverse_project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [Tech Stack](#tech-stack)
+- [👥 Authors](#authors)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse_Project] <a name="about-project"></a>
+
+**[Hello Microverse_Project]** is my first project with Microverse 😀
+
+<!-- TECH STACK -->
+
+### Tech Stack <a name="tech-stack"></a>
+
+> HTML, CSS3 and JS.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+# Juan Sanchez
+
+- GitHub: [@githubhandle](https://github.com/J-C-S-V)
+- Twitter: [@twitterhandle](https://twitter.com/juansan0)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/juan-carlos-sanchez-vargas-a308b014b/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give a star!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] :+1: You have made a brilliant effort to get this up and running you still need to give your nice README.md some more time and attention so that you can have content that make your project looks professional to visitors or users. Kindly check [this README template](https://github.com/microverseinc/readme-template) again and see all the important sub-titles that you are missing and their respective details.
  - [ ] * The only **optional** section is FAQ.
  * You can delete the live demo section since you are not deploying your application now.
  * Kindly add the section **Future Features**, including 1-3 potential features to be added.
* It will be more customised to **remove the section description**, i.e. "> Describe the tech stack and include only the relevant sections that apply to your project." should be removed and filled in with their custom content.

Reply:
Hello Abayomi,

Thanks for the feedback. Live demo was deleted and I added the section **Future Features**. The **description section** was not present before so I didn't have to remove it.

Finally I kept all non skipeable titles and subtitles.

Regards!

Reply:
Oh, I'm sorry about that chief :praying, I literally copy my previous comment for a different Micronaut in order to provide a prompt review. Sorry about that, brother.